### PR TITLE
[Pomf.se] Added a.pomf.se

### PR DESCRIPTION
**Pull request by StevenRoddis**

### CODE REVIEW

File: src/chrome/content/rules/Pomf.se.xml [hunk 1]
```diff
@@ -15,18 +15,28 @@
...
 
 	<securecookie host="^\.pomf\.se$" name="^__cfduid$" />
 
 
-	<rule from="^http://(?:www\.)?pomf\.se/"
-		to="https://pomf.se/" />
+	<rule from="^http://(www\.)?pomf\.se/"
+			to="https://pomf.se/" />
 
+	<test url="http://pomf.se/" />
```
Comment (jsha) [inline src/chrome/content/rules/Pomf.se.xml]: These are redundant since they are automatically created by the target tag, so you can leave them out.

**Comment by StevenRoddis** [comment]
Done

File: src/chrome/content/rules/Pomf.se.xml (hunk 1 above)
Comment (jsha) [inline src/chrome/content/rules/Pomf.se.xml]: You missed these two. :-)

**Comment by jsha** [comment]
FYI (since the comment structure doesn't make it clear): I'm waiting for this branch on the deletion of the two remaining redundant test URLs.

**Comment by StevenRoddis** [comment]
Cheers, should be good to go.

**Pull request merged**